workspace-service/src/graphql/validation.rs

```diff
@@ -0,0 +1,41 @@
+use std::fmt::{Display, Formatter, Result};
+use validator::{ValidationErrors, ValidationErrorsKind};
+
+const SEPARATOR: &str = ", ";
+
+pub struct ValidationError(ValidationErrors);
+
+impl Display for ValidationError {
+    fn fmt(&self, f: &mut Formatter<'_>) -> Result {
+        let errors = self.0.clone().into_errors();
+        let errors = errors.iter().map(|(_, v)| v);
```

Comment:
Not sure if we have to clone here since we clone the individual errors later already. The `HashMap` also has a shorthand to get a list of values. So we can replace these 2 lines with:
```suggestion
        let errors = self.0.errors().values();
```

Reply:
nice! Thanks